Sources/Swagger/Operation.swift

```diff
@@ -11,9 +11,9 @@ public struct Operation {
     public let operationParameters: [PossibleReference<Parameter>]
 
     public var parameters: [PossibleReference<Parameter>] {
-        return operationParameters + pathParameters.filter { pathParam in
-                operationParameters.contains { $0.value.name == pathParam.name }
-        }
+        return pathParameters.filter { pathParam in
+            !operationParameters.contains { $0.value.name == pathParam.value.name }
+        } + operationParameters
```

Comment:
This is to show "shared between operations" parameters first (i.e. the path parameters not overridden at the operation level).
This means that for `/pets/{petId}`, the `petId` parameter would be first in the list of parameters if not overridden.